torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -1026,7 +1029,7 @@ def test_average_parameters(self):
                 nn.Conv2d(3, 3, kernel_size=3, padding=1),
                 nn.ReLU(),
                 nn.Linear(1, 5, bias=False),
-            ).cuda(device_id)
+            ).to(torch.device(device_id))
```

Comment:
.to(torch.accelerator.current_accelerator().type + ":" + str(device_id))

Reply:
no need